apps/widget/package.json

```
@@ -11,6 +11,7 @@
     "envsetup:docker": "chmod +x ./env.sh && ./env.sh && mv ./env-config.js ./build/env-config.js",
     "start:static:build": "pnpm envsetup:docker && http-server build -p 4500 --proxy http://localhost:4500?",
     "start:docker": "pnpm build && pnpm start:static:build",
+    "docker:build": "docker build -f ./Dockerfile -t novu-widget ./../..",
```

```suggestion
"docker:build": "pnpm --silent --workspace-root pnpm-context -- apps/widget/Dockerfile | docker buildx build --build-arg PACKAGE_PATH=apps/widget - -t novu-widget",```

Same reasoning as for Web app. Though I might start to suspect you are leaving the UI apps with a different build type for a specific reason I can't guess right now. 🤔

---

@p-fernandez you included your comment into the the code snippet